pandas/core/series.py

```diff
@@ -2683,13 +2683,18 @@ def quantile(
             This optional parameter specifies the interpolation method to use,
             when the desired quantile lies between two data points `i` and `j`:
 
-                * linear: `i + (j - i) * fraction`, where `fraction` is the
-                  fractional part of the index surrounded by `i` and `j`.
+                * linear: `i + (j - i) * fraction`, where `fraction` is the proportion
+                of the distance between `i` and `j`. it refers to the relative position
+                of the desired quantile value between i and j
+                hence fraction = (desired_quantile - i) / (j - i)
                 * lower: `i`.
                 * higher: `j`.
                 * nearest: `i` or `j` whichever is nearest.
                 * midpoint: (`i` + `j`) / 2.
 
+            For example, if (len(self) - 1) * q is 9.6, and the elements
```

Comment:
Can you add this as a code example in the `Examples` section instead?

Reply:
Do I add it the same way it currently is as an explanation?

Reply:
You can create a `Series` with data that follows this example

Reply:
Alright. Thank you